### ISSUE TITLE
Rename file with extension

### ISSUE THREAD

**Issue by valera911** [geberl/swift-localstorage#26]
Hello. 
I know the app is representing like not file manager but current it's impassible to change a file extension from the apple file app. 
It would be very useful to have a changing extension feature, if it wound't break your vision of the app.

**Comment by geberl** [comment]
Yes, I agree. This would be a feature that fits nicely with Local Storage.
Will implement when I got time.
